src/main/java/frc/robot/RobotContainer.java

```diff
@@ -6,9 +6,8 @@
 
 import edu.wpi.first.wpilibj.GenericHID;
 import edu.wpi.first.wpilibj.XboxController;
-import frc.robot.commands.ExampleCommand;
-import frc.robot.subsystems.ExampleSubsystem;
 import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.subsystems.*;
```

Comment:
We should ideally list out the subsystems instead of relying on *, just to be a bit more clear.